nautobot/core/views/mixins.py

```diff
@@ -1420,3 +1428,35 @@ def notes(self, request, *args, **kwargs):
             "active_tab": "notes",
         }
         return Response(data)
+
+
+class ObjectDataComplianceViewMixin(NautobotViewSetMixin):
+    """
+    UI Mixin for an DataCompliance for this object.
+
+    base_template: Specify to explicitly identify the base object detail template to render.
+        If not provided, "<app>/<model>.html", "<app>/<model>_retrieve.html", or "generic/object_retrieve.html"
+        will be used, as per `get_base_template()`.
+    """
+
+    base_template: Optional[str] = None
+
+    @drf_action(detail=True)
+    def data_compliance(self, request, *args, **kwargs):
+        model = self.get_queryset().model
+        instance = self.get_object()
+        if not self.queryset:
+            self.queryset = global_apps.get_model(model).objects.all()
```

Comment:
unused?

Reply:
Should have pointed out that this was a carry over from:

https://github.com/nautobot/nautobot/pull/7722/files#diff-328e99e4fba15cca3e4d00b741d71d68ad2d3b44ce5d41587ec45b8b2ff57106L180

Was concerned it was covering a behavior I had not considered.